Makefile

```diff
@@ -84,12 +84,14 @@ check: python-env
 .PHONY: check-headers
 check-headers:
 	@go get github.com/elastic/go-licenser
-	@go-licenser -d
+	@go-licenser -d -exclude vendor -exclude .git -exclude x-pack
+	@go-licenser -d -exclude vendor -license Elastic x-pack
 
 .PHONY: add-headers
 add-headers:
 	@go get github.com/elastic/go-licenser
-	@go-licenser
+	@go-licenser -exclude vendor -exclude .git -exclude x-pack
+	@go-licenser -exclude vendor -license Elastic x-pack
```

Comment:
No need to exclude `vendor` or `.git` its in the default from go licenser.

https://github.com/elastic/go-licenser/blob/02324788cd84244b695b57aec2a4c388ea2d8fc8/main.go#L92